packages/front-end/services/experiments.ts

```diff
@@ -180,7 +180,8 @@ export function applyMetricOverrides<T extends ExperimentMetricInterface>(
       overrideFields.push("windowHours");
     }
     if (!isNil(metricOverride?.delayHours)) {
```

Comment:
I don't think this applies since I didn't change the logic in the overrides themselves since they have such little space they remain in hours. So this isn't really a migration. If the override exists it should override the delayValue.